app.js

```diff
@@ -10,9 +10,7 @@ const {
   limitText
 } = require('./libraries/helpers');
 const generateHTML = require('./libraries/html-generator');
-const {
-  uploadFile
-} = require('./libraries/bucketManager');
+const { uploadFile } = require('./libraries/bucketManager');
```

Comment:
which clean code standart are we using?  why are you modifiying some into one line but leaving out the others?